pkgs/applications/graphics/xournal/default.nix

```diff
@@ -50,8 +54,8 @@ stdenv.mkDerivation rec {
   meta = {
```

Comment:
You can use `meta = with stdenv.lib; ...` to get the attributes like licenses into scope here.